src/styles/blocs/card.scss

```diff
@@ -0,0 +1,112 @@
+.catalog__card {
+  box-sizing: border-box;
+  width: 200px;
+  border: 1px solid #F3F3F3;
+  border-radius: 5px;
+  display: flex;
+  flex-direction: column;
+  padding-inline: 16px;
+  padding-top: 32px;
+  padding-bottom: 16px;
+
+  &-img {
+    height: 134px;
+    width: 160px;
+    display: block;
+    margin: 0 auto 40px;
+    flex-shrink: 0;
+  }
+
+  &-title {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color:#060B35;
+    margin: 0;
+    margin-bottom: 4px;
+  }
+
+  &-product-code {
+    margin: 0;
+    font-size: 10px;
+    line-height: 14px;
+    color: $secondary;
+    margin-bottom: 16px;
+  }
+
+  &-stars-and-reviews {
+    display: flex;
+    margin: 0;
+    margin-bottom: 24px;
+    justify-content: space-between;
+  }
+
+  &-reviews {
+    font-weight: 400;
+    height: 16px;
+    font-size: 10px;
+    line-height: 14px;
+    color:$main-color;
+    display: flex;
+    justify-content: end;
+    align-items: flex-end;
+  }
+
+  &-price {
+    margin: 0;
+    margin-bottom: 16px;
+    display: flex;
+    justify-content: space-between;
+
+    &-name {
+      font-size: 12px;
+      font-weight: 400;
+      line-height: 18px;
+      color: $secondary;
+    }
+
+    &-value {
+      font-size: 16px;
+      font-weight: 700;
+      line-height: 18px;
+      color: $main-color;
+      display: flex;
+      justify-content: right;
+    }
+  }
+
+  &:hover {
+    transform: scale(1.2);
+    transition: all 300ms ease;
+  }
+
+  &:hover .catalog__card-title {
+    transition: all 300ms;
+    color: #34568b;
+  }
+
+  &-button {
+    text-decoration: none;
+    border: 1px solid $blue-accent;
+    background-color: $blue-accent;
+    padding: 11px;
+    border-radius: 5px;
+    color: $white;
+    font-weight: 700;
+    font-size: 14px;
+    line-height: 16px;
+    text-transform: uppercase;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    box-sizing: border-box;
+
+    &:hover {
+      transition: all 300ms;
```

Comment:
Make both 'hover' and 'unhover' be smooth. 
To reach this effect you should apply `transition` not to `:hover` but to static selector
Wrong Example: 
```
.button {
  color: #ff000;
  
  &:hover {
    color: #00ff00;
    transition: color 250ms;
  }
}
```

Correct Example:
```
.button {
  color: #ff000;
  transition: color 250ms;
      
  &:hover {
    color: #00ff00;
  }
}
```

```suggestion
```